src/tests/fs_tests.rs

```diff
@@ -4034,6 +4034,33 @@ pub mod fs_tests {
 
         assert_eq!(cage.close_syscall(fd), 0);
         assert_eq!(cage.exit_syscall(EXIT_SUCCESS), EXIT_SUCCESS);
+
+        lindrustfinalize();
+    }
+
+    pub fn ut_lind_fs_shmget_syscall(){
+        // acquire locks and start env cleanup
+        let _thelock = setup::lock_and_init();
+        let cage = interface::cagetable_getref(1); 
+
+        let key = 33123;
+        // Get shmid of a memory segment / create a new one if it doesn't exist
+        let shmid = cage.shmget_syscall(33123, 1024, IPC_CREAT);       
+        assert_eq!(shmid,4); 
+
+        // Check error upon asking for a valid key and passing the IPC_CREAT or IPC_EXCL flag
```

Comment:
you are passing both flags, not one of them, so I think "IPC_CREAT or IPC_EXCL flag" should be changed to "both IPC_CREAT and IPC_EXCL flags" for clarity.

Reply:
Done!